exec/cnex/nstring.c

```diff
@@ -208,6 +222,99 @@ size_t string_findChar(TString *self, char c)
     return NPOS;
 }
 
+int64_t string_findFirstOf(TString *self, size_t pos, const char *str)
+{
+    if (self->data == NULL || self->length == 0 || pos > self->length || str == NULL) {
+        return -1;
+    }
+
+    for (size_t l = pos; l < self->length; l++) {
+        if (strchr(str, self->data[l]) != NULL) {
+            return l;
+        }
+    }
+    return -1;
+}
+
+static inline BOOL isNotOneOf(TString *self, char c)
+{
+    for (size_t i = 0; i < self->length; i++) {
+        if (self->data[i] == c) {
+            return FALSE;
+        }
+    }
+    return TRUE;
+}
+
+int64_t string_findFirstNotOf(TString *self, size_t pos, TString *p)
```

Comment:
This interface is still inconsistent with `string_findFirstOf`. Why does `string_findFirstOf` take a `const char *`, and `string_findFirstNotOf` take a `TString *`? All these functions should take the same types, whatever you decide.

Reply:
Fixed interface to both take TString's.